src/eddn/conf/Settings.py

```diff
@@ -94,6 +97,8 @@ class _Settings(object):
         "http://schemas.elite-markets.net/eddn/blackmarket/1/test",
         "http://schemas.elite-markets.net/eddn/journal/1",
         "http://schemas.elite-markets.net/eddn/journal/1/test",
+        "http://schemas.elite-markets.net/eddn/fssallbodiesfound/1",
```

Comment:
No need to list any new schemas here.  It's purely so as to reject anyone still (how?) trying to send using the original-hostname schema URLs.